### DISCUSSION TITLE
OTG to Win11

### DISCUSSION THREAD

**Discussion by piekarzma** [WLAN-Pi/feedback#78] [Hardware - WLAN Pi R4] [answered]
Hi there, is there a way to connect WLANPi R4 to OTG via usb to Win11. Win sees WLNPI on serial port com6 not as network adapter. I have established SSH with WLANPI and see interface usb0 is up.

**Comment by joshschmelzle** [comment]
You will need to update the driver for that device with an RNDIS driver.

**Reply by joshschmelzle** [reply, marked as answer]
Driver and instructions here: https://github.com/WLAN-Pi/drivers/blob/main/RNDIS.md
